src/components/utils/ControlledAlert/useAlertStyles.tsx

```diff
@@ -1,66 +1,115 @@
 import { ControlledAlertProps } from './ControlledAlert';
-import { addOpacity } from '../../../utils/helpers';
+import { addOpacity, lightenDarkenColor } from '../../../utils/helpers';
 import useTheme from '../../../utils/useTheme';
+import { Theme } from '../../../theme';
 
 type UseControlledAlertStylesProps = Pick<
   ControlledAlertProps,
   'variant' | 'variantBackgroundStyle'
 >;
 type AlertVariant = UseControlledAlertStylesProps['variant'];
+type ThemeColor = keyof Theme['colors'];
 
-const getAlertVariantColor = (alertVariant: AlertVariant) => {
+const getSolidAlertThemes = (alertVariant: AlertVariant): VariantTheming => {
+  const theme = useTheme();
+  const getBackgroundColor = (variantColor: ThemeColor) => {
+    return lightenDarkenColor(theme.colors[variantColor], 190) as ThemeColor; // TODO: This is as close as I could get to the proper color, and it's a very yellow green...
+  };
```

Comment:
With old styles, where I hard-coded the color:
![SideSheet](https://user-images.githubusercontent.com/10763539/188020934-9caf135e-1690-455b-827b-2f1fad3bfb41.png)
When I try to use the lightenDarkenColor (it keeps getting yellower, not lighter. When I put in 10, or 90 to match "10% Opacity", it's dark green. I think I'm using it wrong:
![Everything went perfectly](https://user-images.githubusercontent.com/10763539/188020980-f7655491-5316-442e-a6b2-3edf1d0d2972.png)

Reply:
Woah, this is interesting, sounds like `lightenDarkenColor` is _**not**_ the way to go! 😱 

I still think there may be a workaround on the color so that we don't have to add a new one to the design system, perhaps we can mix it with white or something like that. Happy to pair program tomorrow on that with ya if that helps move this forward!